### DISCUSSION TITLE
Copilot suggesting 1 line at a time

### DISCUSSION THREAD

**Discussion by tiwazs** [community/community#31398] [Copilot Conversations]
I just started using Copilot but it suggests 1 line at a time instead of the whole block of code. 
searching through the internet I seems that this happened some time ago but was fixed
https://github.com/community/community/discussions/13303

However I'ts not working for me. what could It be?

I tried both copilot and copilot nightly. same problem.

VS code version 1.70.2

**Comment by davecheney** [comment]
@damsog i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Reply by tiwazs** [reply]
Thank you for answering. 
Sorry, I was having that problem yesterday but some minutes later it was working. It happened usually when I'd describe what I wanted it to do in a comment. However It's working fine now, I think I'm just getting used to it.

**Comment by Ganitzsh** [comment]
Hey there, I am still having this issue, but only with JS/TS.

Here in TypeScript I can tell copilot wants to give me a multi-line array but doesn't show more than the first `[` :
<img width="328" alt="Screenshot 2022-10-02 at 13 36 07" src="https://user-images.githubusercontent.com/2302338/193452575-85a7ec2f-356b-4f38-ac14-0deebd1fe9e4.png">


Here is the same example but with Lua:
<img width="954" alt="Screenshot 2022-10-02 at 13 35 13" src="https://user-images.githubusercontent.com/2302338/193452544-c0f6b08a-bdde-4c8d-b248-4a677830b5a9.png">

In Lua no problem, but TypeScript does not work.

**Reply by reazn** [reply]
Same problem here, it's working for all file extensions except js/ts. Did you ever get this issue resolved?

**Reply by Ganitzsh** [reply]
Unfortunately the issue is still here, I contacted the support about it. Here is a link to an issue where I explain more in depth: https://github.com/zbirenbaum/copilot-cmp/issues/10 @REAZN
